.travis.yml

```diff
@@ -34,6 +36,12 @@ jobs:
         - travis_retry wget https://scrutinizer-ci.com/ocular.phar
         - travis_retry php ocular.phar code-coverage:upload --format=php-clover build/logs/clover.xml
 
+    - stage: Test
+      php: nightly
+      before_install:
+        - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
+        - composer config platform.php 7.4.99
```

Comment:
`composer validate --strict` should be moved to `before_install` I think, it prevents you from moving forward (in the php 8 job)